modules/alloy/src/main/ts/ephox/alloy/api/ui/TieredMenu.ts

```diff
@@ -51,7 +53,17 @@ const tieredMenu: TieredMenuSketcher = single<TieredMenuSpecType, TieredMenuDeta
     ]),
 
     FieldSchema.defaulted('fakeFocus', false),
+
+    // DEPRECATE. I don't think it is used anywhere (2022-08-16). We want
```

Comment:
If this is deprecated then we should add a `Deprecated` changelog entry. We should also add `@deprecated` comment on the types so it shows in IDEs (see the followign for an example): https://github.com/tinymce/tinymce/blob/3865a8cb3420afde40926a62680059d046cf28cb/modules/agar/src/main/ts/ephox/agar/api/Chain.ts#L141-L144

Reply:
Actually if we're doing a breaking change, then should we just remove it entirely anyways?

Reply:
I was worried about making too many unrelated changes in this PR that was already pretty big. But if we're happy that even if remark or clerk are using this, because they lock their version, we're happy to do changes later if required, then I can remove it altogether.

Reply:
- [ ] remove onHighlight altogether.

Reply:
Updated: removed onHighlight.